### ISSUE TITLE
Websites in Bandcamp.com aren't auto-defaulting to the container, only the specific page within bandcamp

### ISSUE THREAD

**Issue by davidalayachew** [mozilla/multi-account-containers#2625]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

1. Create a new container for Bandcamp
2. Go to bandcamp.com
3. Click the Always open in this container button to associate bandcamp.com with the container
4. Open up a new tab that is not associated to any containers
5. Go to the following bandcamp link -- https://blancmanioclabel.bandcamp.com/album/joola-jazz?from=hpnn

### Actual behavior

You will see that it opens up as is, without prompting to open in the new Bandcamp container

### Expected behavior

It should prompt you to open in the Bandcamp container

### Additional informations

Firefox has this feature where (if I am not mistaken) the domain name is highlighted when on a site. So for example, on github.com, here is how the url looks.

![image](https://github.com/mozilla/multi-account-containers/assets/38477640/dac093fa-7fe9-4ef2-81a5-4745ac6d3401)

See how github.com is highlighted?

Same thing occurs for bandcamp.com. Here are screenshots, for both the main site, and the link to the album from the recreation steps.

![image](https://github.com/mozilla/multi-account-containers/assets/38477640/c9ca050d-cdd3-4e35-9086-830569ef9b87)

![image](https://github.com/mozilla/multi-account-containers/assets/38477640/ce5540a5-365c-41dc-bf40-ba41291a6ee2)

So, Firefox is clearly aware of the fact that they are from the same domain, even if they are 2 very different links.

I would think then that this would allow them to be under the same container, but alas, that seems not to be the case.

I feel like not auto-defaulting would be bad because many websites are designed in this way. Hell, even github itself does this, with the some_name.github.io format.

**Comment by PrivacyFriendlyMuffins** [comment]
I think I'm having the same issue, if I'm understanding your issue correctly. It seems to be a problem regardless of the artist-specific subdomain. It's happened on multiple different artists' pages for me, and all on Bandcamp.

Anyone have any update on this?

**Comment by davidalayachew** [comment]
@PrivacyFriendlyMuffins No updates thus far. Still waiting myself. I've ended up just manually adding the pages that I listen to to the container of choice. Still not a good situation, but development on this repo is progressing.

In the meantime, I am pretty sure that there is a separate plugin, not developed by Mozilla, that more or less fixes this, as well as adds a whole bunch more functionality. If you don't want to wait for this, that would be where I would check next. I forget the name, sadly.

**Comment by achernyakevich-sc** [comment]
> So, Firefox is clearly aware of the fact that they are from the same domain, even if they are 2 very different links.

This is wrong assumption - technically `blancmanioclabel.bandcamp.com` and `bandcamp.com` are two different domains. The first one is a third level domain, the second one is a second level domain. Similar to like subdomains of `com.au` are completely independent domains representing completely independent companies.

As far as I know there at least two issues in this repository that being implemented would help to handle your use case:
* #473
* #691 

As well there is a separated Firefox add-on that implements required functionality and works well with MAC - [Containerise](https://addons.mozilla.org/addon/containerise/)

I hope this information helped you. So I would propose you to close this issue to avoid duplicates and this way showing other that solution is inside comments. :)

**Comment by davidalayachew** [comment]
Thanks, I'm happy to close this as a duplicate.
